modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/db/Migration.scala

```diff
@@ -206,7 +206,7 @@ object Migration {
           .tableExists[F](goodTableName)
           .ifM(createMissingRecoveryTables, createMissingRecoveryTables.map(_.::(target.createTable(goodModel))))
 
-        createTables.flatMap(blocks => optUpdateGoodTable.map(_.fold(blocks)(blocks :+ _)))
+        optUpdateGoodTable.flatMap(_.fold(createTables)(ugt => createTables.map(_ :+ ugt)))
```

Comment:
@oguzhanunlu this is just a stylistic suggestion, so you're OK to ignore it.  But I would find this line easier to read if it was instead written like:

```scala
  for {
    updates <- optUpdateGoodTable
    creates <- createMissingRecoveryTables
  } yield creates :++ updates
```